resource/modules/resource_match.cpp

```diff
@@ -1666,6 +1666,8 @@ static int run (std::shared_ptr<resource_ctx_t> &ctx,
             rc = tr.run (j, ctx->writers, match_op_t::MATCH_ALLOCATE_W_SATISFIABILITY, jobid, at);
         else if (std::string ("allocate_orelse_reserve") == cmd)
             rc = tr.run (j, ctx->writers, match_op_t::MATCH_ALLOCATE_ORELSE_RESERVE, jobid, at);
+        else if (std::string ("allocate_orelse_reserve") == cmd)
```

Comment:
How would we get to this condition? Should this be like:

```suggestion
        else if (std::string ("grow_allocate") == cmd)
```

Reply:
And minor suggestion to make it ` match_op_t::MATCH_GROW_ALLOCATE` just to match the naming convention of the others (action/verb "allocate" vs noun "allocation")

Reply:
> How would we get to this condition? Should this be like:

That's an error in my copy-paste. Fixed.

Reply:
> And minor suggestion to make it  match_op_t::MATCH_GROW_ALLOCATE just to match the naming convention of the others (action/verb "allocate" vs noun "allocation")

I considered both options and am leaning toward `match_op_t::MATCH_GROW_ALLOCATION` and `"grow_allocation"` given the precedent of `match_op_t::MATCH_ALLOCATE_W_SATISFIABILITY`.